### DISCUSSION TITLE
Add "Fullscreen" to toolbar

### DISCUSSION THREAD

**Discussion by NvnStar** [Floorp-Projects/Floorp#596] [Feature Request]
Would be nice if we have the Fullscreen option in the customize toolbar option.

**Comment by creeper-0910** [comment]
Is it more convenient than the F11 shortcut?

**Reply by NvnStar** [reply]
I would say yes, always been using that in FF. There will be time that you only use mouse when browsing, having fullscreen at toolbar for 1-click really helps.

**Comment by breadbeforerice** [comment]
Firefox has it and it would be nice to have it. I thought it had been moved to the status bar, but it was not there. I am not sure why it is gone.
